docs/install_windows.rst

```diff
@@ -31,6 +31,9 @@ Using PowerShell and Chocolatey (recommended)
 To install via PowerShell, search "powershell" in the Windows start menu,
 right-click on it and then click "Run as administrator"
 
+.. note:: This is the only time you should use administrator during the 
+          installation and usage of Red.
```

Comment:
I think this note would be more successful if it were done in "Creating a virtual environment" section (of course reworded so that it makes sense in there)

Reply:
I think this is good? It looks a bit busy/full, but I'm out of ideas.
![http://i.vex.my.to/6rKkqF](http://i.vex.my.to/6rKkqF)